### ISSUE TITLE
Render tables consistently across site

### ISSUE THREAD

**Issue by Jkrzy** [18F/tock#1058]
As a tock user I want tables styled consistently across the site so that as I check reports across multiple pages I'm not surprised by how the data is being displayed.

The tables rendered on `ReportingPeriodDetailView` are inconsistent and visually difficult to interpret. We'll settle on a reasonable set of styles of tables and implement them throughout the site.

## We'll know we're done when
- [ ] The tables on the `ReportingPeriodDetailView` are styled  consistently with others throughout the site

**Comment by thisisdano** [comment]
Can you post a screenshot of the tables/sections in question? — I could probably then hazard some advice

**Comment by tbaxter-18f** [comment]
@kfoley-18F could you post some shots of what prompted this? As I look at https://tock.18f.gov/employees/timothy.baxter/ and https://tock.18f.gov/utilization/ they look same-ish to me, but I could be missing something.

**Comment by kfoley-18F** [comment]
![Screen Shot 2020-04-22 at 9 20 57 AM](https://user-images.githubusercontent.com/51135391/80007112-afe33100-847a-11ea-9174-074e8bb51f82.png)

Doesn't show the table gridlines for me for any user and it's super hard to read.

**Comment by thisisdano** [comment]
I definitely agree that the lack of gridlines (or other clear distinguishing characteristics between rows) makes this very hard to parse! 

This _could_ just be a job for a simple `usa-table--borderless` variant — perhaps with an adjustment to its cell padding. That solution might also address the smaller issue of the column headings having a different alignment than their content (centered headings, left-aligned content)

But I think with a couple small tweaks, this would show a big improvement

**Comment by tbaxter-18f** [comment]
@kfoley-18F what's the URL for that page, too? I'm wondering if we've got some redundancy.

**Comment by kfoley-18F** [comment]
It's https://tock.18f.gov/reports/2020-04-12/kelsey.foley/
I get to it via: 
Users > scroll to my name, click > get the table of utilization and last 5 weeks, click on a specific date > get to this page. 

I've been using it often to dig in on things when I need to see the written snippets for certain time codes and not just the hours/codes.

**Comment by timoballard** [comment]
Looking for feedback on the first pass at this table cleanup:

![image](https://user-images.githubusercontent.com/1425377/80498945-f7ae0080-8931-11ea-9e17-781b9f7bfee8.png)

**Comment by tbaxter-18f** [comment]
Looks good to me!
